EasyPost/Request.cs

```diff
@@ -103,7 +103,7 @@ public class Request {
                         result.AddRange(FlattenParameters(list[i], string.Concat(parent, "[", pair.Key, "][", i, "]")));
                     }
                 } else if (pair.Value is DateTime) {
-                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), Convert.ToString(((DateTime)pair.Value).ToString("yyyy-MM-ddTHH:mm:sszzz"))));
+                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), ((DateTime)pair.Value).ToLocalTime().ToString("yyyy-MM-ddTHH:mm:sszzz")));
```

Comment:
Hi Andrew,

if you debug this
`new` DateTime(636210240270000000, DateTimeKind.Utc).ToString("yyyy-MM-ddTHH:mm:sszzz");
you'll se just a warning, so you can't run a test that fails unless the test consider a warning as an error.
But, in my humble opinion, you can fix and prevent the warning.